free-1.md

```diff
@@ -57,12 +105,26 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I think this is very much a problem about langauge design and not so implementation
+heavy, especially for just describing and searching up hands.  If we wanted to
+add features for analyzing hands I think that would increase the "system" aspects
+of the project by a large amount, so perhaps that would be better to not implement
+for a CS 111 project and just design how that part would work if we were to
+implement it.  I think it would still be very useful without analysis features.
+
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+I think this is not super amibitious.  Some of the analysis features could be
+harder, but searching for things and designing an easy to read format should not
+be too difficult I don't think.
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+I think this is a very achievable project in a relatively short time frame, which
+I think makes it a good candidate for a CS111 project.  On the other hand I'm not
+sure if other people like bridge as much as I do, so it might be harder to find
+classmates who want to work with me on it and are passionate about it.
```

Comment:
It's good to do a project about something you like. It's fine if you're the only one in the class who likes it :). But you will have to explain and motivate the domain well enough so that other people in the class can understand and help you with it (via critique).